checks.py

```diff
@@ -12,6 +12,7 @@
 WEBSITE_DUPLICATED_PROTOCOL_RE = re.compile(r'^\w{2,}://\w{2,}://')
 WEBSITE_SHORTENER_RE = re.compile(r'^\w{2,}://(www\.)?(tinyurl\.com|tiny\.(cc|pl)|(bit|cutt)\.ly|[gt]\.co|goo\.gl(?!/maps))/',
                                   re.IGNORECASE)
+STREET_NAME_BAD_PREFIX = re.compile(r'^ul(ica)?\.? ')
```

Comment:
```suggestion
STREET_NAME_BAD_PREFIX = re.compile(r'^ul(ica)?\.? ', re.IGNORECASE)
```

Przy okazji można wychwycić Ul. UL. itp.

Reply:
Nie mam pojęcia czemu o tym zapomniałem, oczywiście masz rację.